### ISSUE TITLE
[Bug report] Typos in options

### ISSUE THREAD

**Issue by Mister-Hope** [Zhengqbbb/cz-git#82]
### ✅ Checklist

- [X] I am using the **latest version**
- [X] I have searched [FAQ](https://cz-git.qbb.sh/faq/)

### 📄 System Info

```shell
Not related
```

### 🔎 Description

I am not sure if you realize there are mispelling in your options.

`allowCustomIssuePrefixs` and `allowEmptyIssuePrefixs` confuse native english speakers.

The options should be `allowCustomIssuePrefix` and `allowEmptyIssuePrefix`, which do not need to be plural.

If you DO PREFER plural, prefix should be `prefixes`.

**Comment by Mister-Hope** [comment]
If you do not want v2 for this change, mark the original one as deprecated and support the correct spelling, thanks.

**Comment by Zhengqbbb** [comment]
ref: https://github.com/Zhengqbbb/cz-git/pull/36

I will fix the option typos in v1.3.13.
Thx u report and great suggest.

**Comment by Zhengqbbb** [comment]
The latest version 🌟(v1.4.0)🌟 should be resolve this issue request , please have a try. 💪
👀 Let me know if there are still any issues. Thanks for you suggestion! 🎉


* **cz-git,czg:** main fix `issuePrefixs` typo field
  * Current version has been already processed for **normal compatibility**

```diff
{
  message: {
-   footerPrefixsSelect
+   footerPrefixesSelect

-   customFooterPrefixs
+   customFooterPrefix
 }

-   issuePrefixs
+   issuePrefixes

-   customIssuePrefixsAlign
+   customIssuePrefixAlign

-   emptyIssuePrefixsAlias
+   emptyIssuePrefixAlias

-   customIssuePrefixsAlias
+   customIssuePrefixAlias

-   allowCustomIssuePrefixs
+   allowCustomIssuePrefix

-   allowEmptyIssuePrefixs
+   allowEmptyIssuePrefix
}
```
